browser/src/control/Control.UIManager.js

```diff
@@ -1710,6 +1710,9 @@ L.Control.UIManager = L.Control.extend({
 	enableTooltip: function(element) {
 		var elem = $(element);
 		if (window.mode.isDesktop()) {
+			if($('.ui-tooltip').length > 0)
+				$(".ui-tooltip").remove();
+
 			elem.tooltip();
```

Comment:
will it work to save result in variable and later just use it to close using an JS API?

```
if (this._tooltip) {
    this._tooltip.tooltip('close');
    this._tooltip = undefined;
}
this._tooltip = elem.tooltip();
```